runtime/js/99_main.js

```diff
@@ -859,7 +859,12 @@ function bootstrapMainRuntime(runtimeOptions, warmup = false) {
     ObjectDefineProperty(globalThis, "Deno", core.propReadOnly(finalDenoNs));
 
     if (nodeBootstrap) {
-      nodeBootstrap(hasNodeModulesDir, argv0, /* runningOnMainThread */ true);
+      nodeBootstrap({
+        usesLocalNodeModulesDir: hasNodeModulesDir,
+        runningOnMainThread: true,
+        argv0,
+        nodeDebug: Deno.env.get("NODE_DEBUG"),
```

Comment:
That's gonna result in the same problem, just the call became hoisted to other place. We should read this value in Rust when we create a worker and pass it in "bootstrap args". Could you change that?

Reply:
Thanks for the pointer - updated it.